### ISSUE TITLE
[prometheus] [question] Blackbox Exporter Usage

### ISSUE THREAD

**Issue by con5cience** [prometheus-community/helm-charts#1809]
Hi there,

It looks like the `prometheus` Helm chart already has the Blackbox Exporter installed and configured to scrape internal cluster services with a specific annotation. That's great!

However, I'd also like to scrape external HTTP endpoints outside of the cluster. How do I:
* Configure the bundled Blackbox Exporter to scrape external endpoints, and
* Configure additional probers beyond `http_2xx`?
 
A sample complex use case would be checking GraphQL endpoints, passing a pre-formatted request body with headers, checking the response body for a string, etc.

I'm not sure if I should also deploy the `prometheus-blackbox-exporter` chart and do all the prober configuration there, then refer to the probers in the `prometheus` chart's values in the `extraScrapeConfigs` section?

Any additional context would be greatly appreciated. I don't have strong attachments to how this shakes out (what charts, where configs go, etc) as long as the end result is achieved. :) Thanks in advance!

**Comment by con5cience** [comment]
Got this working.

tl;dr yes, must deploy `prometheus-blackbox-exporter` chart and configure probers there, and reference them in the `prometheus` chart with a scraper job.

**Comment by OneCricketeer** [comment]
> and configure probers there

Could you explain this in more detail? I do see the chart values have `target` list there, but there is also `Probe` CRD that can reference the blackbox exporter service like so 

```yml
apiVersion: monitoring.coreos.com/v1
kind: Probe
metadata:
  name: example-probe
  namespace: 'default'
  labels: {}
spec:
  jobName: http-probe-job
  interval: 60s
  module: http_2xx
  prober:
    url: prometheus-blackbox-exporter.svc:9115
    scheme: http
    path: /probe
  targets:
    staticConfig:
      static:
        - https://github.com
```

Otherwise, [target list defines `ServiceMonitor`'s for each target](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-blackbox-exporter/templates/servicemonitor.yaml#L2-L31), which I guess achieves a similar goal, but depending on resource limits on the k8s cluster, one Probe with N targets is probably better than N ServiceMonitors
